system-test/src/systemTest/java/org/apache/mesos/elasticsearch/systemtest/DiscoverySystemTest.java

```diff
@@ -25,6 +28,23 @@ public void testNodeDiscoveryRest() throws InterruptedException {
 
         ElasticsearchNodesResponse nodesResponse = new ElasticsearchNodesResponse(tasks, NODE_COUNT);
         assertTrue("Elasticsearch nodes did not discover each other within 5 minutes", nodesResponse.isDiscoverySuccessful());
-    }
+
```

Comment:
Is it worth adding another reconcilliation test to test that if an executor dies and restarts, it continues to use the same data files, rather than creating new ones?